metrics/prometheus_reporter.go

```diff
@@ -23,6 +23,7 @@ package metrics
 import (
 	"fmt"
 	"github.com/topfreegames/pitaya/logger"
+	"golang.org/x/xerrors"
```

Comment:
don't think you need this dependency. you should use constants

Reply:
Fixed in latest commit